Makefile

```diff
@@ -53,6 +53,7 @@ format:
 	isort .
 	black --config pyproject.toml .
 	flake8 .
+	mypy .
```

Comment:
Do we need to run `mypy` here?  If so, should we do the same in admin?  `mypy` is currently run as part of `make test` (in `run_scripts.sh`).

Reply:
I would add it in admin as well. its part of scripts/run_tests but wasn't part of `make format` . I generally run `make format` before I push